.typos-config.toml

```diff
@@ -3,4 +3,7 @@ extend-exclude = ["*.pem", "libcaf_net/test/pem.cpp", "Jenkinsfile"]
 
 [default.extend-words]
 caf = "caf"
-CAF = "CAF"
+aout = "aout"
+ws = "ws"
+yout = "yout"
```

Comment:
Typo? I couldn't find a match for `yout` (only for `layout`).

Reply:
That's for the template parameter Yout, e.g..

```
error: `Yout` should be `Your`
  --> ./libcaf_core/caf/interface_mismatch.hpp:77:26
   |
77 |         type_list<Zs..., Yout(Yin...)>> {};
   |                          ^^^^
```

Reply:
Ah, I didn't know it's matching case-insensitive.